zed/CMakeLists.txt

```diff
@@ -17,7 +19,7 @@ SET(EXECUTABLE_OUTPUT_PATH ".")
 
 find_package(ZED 3 REQUIRED)
 find_package(OpenCV REQUIRED)
-find_package(CUDA ${ZED_CUDA_VERSION} EXACT REQUIRED)
```

Comment:
Yeah, let's put it back

Reply:
I changes this to required from exact required because we originally were running with CUDA 11.1, but Ubuntu received an upgrade which broke this compilation. However, as far as I know, CUDA 11.2 should be a compatible release, so I removed 'EXACT'.